src/fileWatcher.ts

```diff
@@ -64,6 +64,8 @@ export namespace ProjectFileWatcher {
                 }
             } else if (uri.fsPath.endsWith('.ibmi.json')) {
                 iProject.setBuildMap(undefined);
+
+                ProjectManager.fire({ type: 'buildMap', iProject });
```

Comment:
so you would not fire an event specific to the ccsid of the particular directory?

Reply:
Thanks for catching that. Updated the PR